boards/arm/frdm_k64f/Kconfig.defconfig

```diff
@@ -43,6 +43,9 @@ config I2C_0
 	default y
 	depends on I2C
 
+config ADC_0
```

Comment:
This doesn't have much meaning for the driver anymore.  (and I'm trying to remove it in PR #24891)

Reply:
Removed.